cicd/forgeops-tests/config/ProductConfig.py

```diff
@@ -0,0 +1,67 @@
+"""
+Product config class for tests. Parses env variables into classes and provides them to tests
+Set env variables to override default ones before running tests
+
+Also provides useful generated variables for products (rest endpoints url, etc...)
+
+"""
+
+import os
+
+
+class AMConfig(object):
+    def __init__(self):
+        """
+
+        """
+        try:
+            self.am_url = os.environ['AM_URL']
+        except KeyError:
+            self.am_url = 'http://openam.pavel.forgeops.com/openam'
+
+        try:
+            self.amadmin_pwd = os.environ['AM_ADMIN_PWD']
+        except KeyError:
+            self.amadmin_pwd = 'password'
+
+        self.rest_authn_url = self.am_url + '/json/authenticate'
+        self.rest_oauth2_authz_url = self.am_url + '/oauth2/authorize'
+
+
+class IDMConfig(object):
+    def __init__(self):
+        try:
+            self.idm_url = os.environ['IDM_URL']
+        except KeyError:
+            self.idm_url = 'http://openidm.pavel.forgeops.com/openidm'
```

Comment:
I assume that later we will factor out all the 'pavel' specific paths?

Reply:
Oh, i will remove them. Good catch. Will go with something generic (openam.example.forgeops.com, etc...)